apps/rush-lib/src/api/RushConfigurationProject.ts

```diff
@@ -72,11 +73,13 @@ export class RushConfigurationProject {
   public constructor(
```

Comment:
These args have gotten complex enough that I think this warrants an `options` object.